CHANGELOG.md

```diff
@@ -144,8 +145,7 @@ Yanked due to critical issue in eslint-module-utils with cache key resulting fro
 - Something horrible happened during `npm prepublish` of 1.10.1.
   Several `rm -rf node_modules && npm i` and `gulp clean && npm prepublish`s later, it is rebuilt and republished as 1.10.2. Thanks [@rhettlivingston] for noticing and reporting!
 
-## [1.10.1] - 2016-07-02 [
-ED]
+## [1.10.1] - 2016-07-02 [YANKED]
```

Comment:
fwiw, I thought `YANKED` was a bit weird also, but I used it because it was explicitly defined on http://keepachangelog.com/ when I started it. I'm fine to move to `UNPUBLISHED` if that's generally well-understood